specificatie/domain.yaml

```diff
@@ -176,9 +176,13 @@ components:
                     Partnerschap is een groep gegevens over de huwelijkse- of partnerschapstatus van een persoon.
       properties:
         datumOntbinding:
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/DatumOnvolledig"
+          allOf:
```

Comment:
depracated: true moet buiten de allOf worden geplaatst, dus

```
          deprecated: true
          allOf:
            - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/DatumOnvolledig"

```

Ik zie nl. in de code generatie dat de huidige declaratie niet leidt tot het deprecated maken van de property in code

Reply:
Aangepast